tools/sklearn_tools/discriminant.xml

```diff
@@ -0,0 +1,199 @@
+<tool id="discriminant_classifier" name="Discriminant Analysis" version="@VERSION@">
+    <description></description>
+    <macros>
+        <token name="@VERSION@">0.9</token>
```

Comment:
Can we move the version macro into the main macro file? So the entire suite gets a main version number?